### DISCUSSION TITLE
ピンヘッダを基盤に半田付けしてしまいました

### DISCUSSION THREAD

**Discussion by yukiyohure** [Yowkees/keyball#606] [Q&A]
Keyballを初めて組み立てている者です。
Keyball 61 組み立て工程の「[ProMicroのはんだ付け](https://github.com/Yowkees/keyball/blob/main/keyball61/doc/rev1/buildguide_jp_pinheader.md#3-6promicro%E3%81%AE%E3%81%AF%E3%82%93%E3%81%A0%E4%BB%98%E3%81%91)」についてです。

ビルドガイドにはピンソケットを基盤にはんだ付けした後、そのピンソケットにピンヘッダを差し込むように指示があります。
その工程で謝って片方基盤だけですが、ピンソケットではなくピンヘッダを基盤にはんだ付けしてしまいました。

慌ててはんだ吸い取り線ではんだを除いてピンヘッダを抜こうとしたのですが、基盤が汚れるだけで抜くことはできませんでした。
この場合、どう対処すれば良いのでしょうか？？

![image](https://github.com/Yowkees/keyball/assets/35441214/208d68f8-4fa0-420d-9a7c-1ee79d60e821)

![image](https://github.com/Yowkees/keyball/assets/35441214/43396c59-934b-422b-9b43-9d6223cb47e2)

**Comment by Rokiomino** [comment]
いざという時に交換できるようにはんだ付けしないだけなので、ひとまずそのまま製作されてはいかがでしょうか？

**Reply by yukiyohure** [reply]
ご回答ありがとうございます！
すでに半田付けしてしまったピンヘッダはそのままに、その上からピンソケットを差し込むことにしました！
